tests/import_test.go

```diff
@@ -2134,6 +2134,67 @@ var _ = Describe("Import populator", func() {
 	})
 })
 
+var _ = Describe("Containerdisk envs to PVC labels", func() {
+	f := framework.NewFramework(namespacePrefix)
+
+	// The corresponding env var is defined in tests/BUILD.bazel (for pullMethod node)
+	// and tools/cdi-func-test-registry-init/populate-registry.sh (for pullMethod pod).
+	const (
+		testKubevirtIoKey   = "test.kubevirt.io/test"
+		testKubevirtIoValue = "testvalue"
+	)
+
+	var (
+		dataVolume          *cdiv1.DataVolume
+		tinyCoreRegistryURL = func() string { return fmt.Sprintf(utils.TinyCoreIsoRegistryURL, f.CdiInstallNs) }
+		trustedRegistryURL  = func() string { return fmt.Sprintf(utils.TrustedRegistryURL, f.DockerPrefix) }
+	)
+
+	AfterEach(func() {
+		By("Delete DV")
+		err := utils.DeleteDataVolume(f.CdiClient, f.Namespace.Name, dataVolume.Name)
+		Expect(err).ToNot(HaveOccurred())
+
+		Eventually(func() bool {
+			_, err := f.K8sClient.CoreV1().PersistentVolumeClaims(f.Namespace.Name).Get(context.TODO(), dataVolume.Name, metav1.GetOptions{})
+			return k8serrors.IsNotFound(err)
+		}, timeout, pollingInterval).Should(BeTrue())
+	})
+
+	DescribeTable("Import should add KUBEVIRT_IO_ env vars to PVC labels when source is registry", func(pullMethod cdiv1.RegistryPullMethod, urlFn func() string) {
```

Comment:
I think we can even integrate the label check into existing tests, but then it would bug me that something this big
doesn't have it's own test.. WDYT

Reply:
I'd keep it separate to be honest. As you said, it's a feature and IMO it warrants its own test.